### ISSUE TITLE
NixOS no option to start service

### ISSUE THREAD

**Issue by rootBrz** [ilya-zlobintsev/LACT#317]
### Checklist

- [X] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#hardware-support) as a known limitation
- [X] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration

### Bug description

There is no way to start systemd service on NixOS, as it should be done through config.
![image](https://github.com/ilya-zlobintsev/LACT/assets/29731178/734db945-4e35-482c-bf83-f59f66ad0612)

For example keyd: 
![image](https://github.com/ilya-zlobintsev/LACT/assets/29731178/91ead42f-1f5a-4bde-a62a-23b43fb65833)

### System info

```markdown
- LACT version: 0.5.4
- GPU model: RX 580 8GB
- Kernel version: 6.6.28 
- Distribution: NixOS Unstable
```

**Comment by ilya-zlobintsev** [comment]
You should report this to the nix package maintainer, it's not packaged here.

**Comment by Atemu** [comment]
Yes, please open an issue in Nixpkgs instead. This one can be closed.

I've written myself a custom module: https://github.com/Atemu/nixos-config/blob/2aedffdf2ec295a60c7b0f27074333d2418ad11c/modules/lact/module.nix that you can use in the meant time. I've been wanting to upstream that now that I tested it a little for myself.

**Comment by JaysFreaky** [comment]
I sent an email to the package maintainer informing them of this thread, so maybe it'll be fixed.
In the meantime, I've added this to my config to get it working:

systemd.services.lactd = {
  after = [ "multi-user.target" ];
  description = "AMDGPU Control Daemon";
  wantedBy = [ "multi-user.target" ];
  serviceConfig = {
    ExecStart = ''${pkgs.lact}/bin/lact daemon'';
    Nice = "-10";
  };
};

**Comment by Atemu** [comment]
Please just create a feature request on the Nixpkgs issue tracker. As a package maintainer myself, I'd also prefer a public discussion rather than an email.

The MVP to get the lact daemon to run are these lines: 

https://github.com/Atemu/nixos-config/blob/2aedffdf2ec295a60c7b0f27074333d2418ad11c/modules/lact/module.nix#L35-L38
